kolibri/core/assets/src/core-app/constructor.js

```diff
@@ -64,6 +66,11 @@ export default class CoreApp {
       mediator.setReady();
     };
 
+    if (process.env.NODE !== 'production') {
+      const colourPicker = require('../utils/colourPicker').default;
+      colourPicker.start();
+    }
```

Comment:
This is fun, but I don't think it should be turned on by default in the devserver.

Also, `colourPicker.js` is unnecessary. You can just drop `ColourPicker.vue` in `CoreBase.vue`.

 Maybe add a prop on `CoreBase` like

```javascript
showColorPicker: {
  type: Boolean,
  default: false,
}
```

and then add to the template

```html
...

    <GlobalSnackbar />
    <ColourPicker v-if="showColorPicker" />
  </div>

</template>
```

Reply:
Yeah, I did it like this so that it wouldn't even get bundled into the production build.